__tests__/input.js

```diff
@@ -0,0 +1,22 @@
+import { By } from 'selenium-webdriver';
+
+import { imageSnapshotOptions, timeouts } from './constants.json';
+
+// selenium-webdriver API doc:
+// https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html
+
+jest.setTimeout(timeouts.test);
+
+test('Textarea input', async () => {
+  const { driver } = await setupWebDriver({
+    styleSet: { sendBoxScrollingTextInput: false }
+  });
+
+  const textarea = await driver.findElement(By.tagName('textarea'));
+
+  await textarea.sendKeys('Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.');
+
+  const base64PNG = await driver.takeScreenshot();
```

Comment:
I would do multiple tests: include one that verifies the resizing of the text area if text has been deleted, and when whitespace has been added.

Reply:
I guess this answers my earlier question.